src/Utils/SchemaPrinter.php

```diff
@@ -157,7 +157,7 @@ protected static function printFilteredSchema(Schema $schema, callable $directiv
             $elements[] = static::printType($type, $options);
         }
 
-        return \implode("\n\n", \array_filter($elements)) . "\n";
+        return \implode("\n\n", \array_filter($elements, fn (?string $element) => $element !== null && $element !== '')) . "\n";
```

Comment:
Also here and in other places, let's keep `array_filter` without callback allowed.